php/libraries/Visit.class.inc

```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * This file contains the Visit class.
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * Base class that contains method to access the visit of projects
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Visit
+{
+    /**
+     * Return an array containing the label and legacyLabel of a visit
+     *
+     * @param integer $id the identifier of the label
+     *
+     * @throws DatabaseException
+     *
+     * @return array a hash with both label
+     *                 hash is of the form
+     *                 array('Label'       => 'label',
+     *                       'LegacyLabel' => 'legacy label')
+     */
+    public function getLabel($id)
```

Comment:
since it's a new class, the parameter and return value should be type hinted.

I also think that `$id` should be in a constructor of a visit that you're working with, and not passed to every call.

And I'm still not sure what a "LegacyLabel" is or why getLabel is returning it.

Reply:
As an end-goal I agree. For the purposes of this PR i think we need to discuss the scope of the changes in order to avoid a very complex review process